apps/fluentd-cloudwatch/daemonset.yaml

```diff
@@ -59,6 +59,12 @@ spec:
             - name: config-volume
               mountPath: /fluentd/etc/conf.d
               readOnly: false
+          livenessProbe:
+            httpGet:
+              path: /metrics
+              port: 9882
```

Comment:
Have you tested this in your sandbox? Where did you get this port number?

It seems that it should be port 24231:
https://github.com/dfds/platform-apps/blob/6a1cb1a88a6fe5aedd6db2b677a14195506829a5/apps/fluentd-cloudwatch/service.yaml#L14-L15
https://github.com/fluent/fluentd-kubernetes-daemonset/blob/master/docker-image/v1.11/debian-cloudwatch/conf/prometheus.conf#L4-L11

Reply:
Changes done, testing again in my sandbox.